### PR TITLE
CH-207: Address the retrospective review comments on the Configuration section

### PR DESCRIPTION
**Pull request by NareshGopal**
# Problem Statement
- Address the retrospective review comments on the Configuration section

# Design
- Some styling fixes and reset functionality would make an API call to get the default values.

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with the PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: manager/gui/src/components/configuration/LrNodeConfiguration.vue [hunk 1]
```diff
@@ -121,12 +122,12 @@
...
                   @click="applyNetworkInfo"
                   :disabled="!isNetworkDetailsValid"
-                  :dark="isFormValid"
+                  :dark="isNetworkDetailsValid"
                   >Apply
                 </v-btn>
                 <v-btn
                   color="csmdisabled"
-                  @click="resetNetworkData"
+                  @click="setNetworkInfoAllNodes(true)"
```
Comment (sujit-h-singh) [inline manager/gui/src/components/configuration/LrNodeConfiguration.vue:130]: you would need a confirmation modal before resetting the data to default

File: manager/gui/src/components/configuration/LrClusterConfiguration.vue [hunk 2]
```diff
@@ -149,14 +153,14 @@
...
-                  color="primary"
-                  @click="applyLogInfo"
-                  :dark="isFormValid"
+                <v-btn class="mr-5" color="primary" @click="applyLogInfo"
                   >Apply
                 </v-btn>
-                <v-btn color="csmdisabled" @click="resetLogData" depressed dark
+                <v-btn
+                  color="csmdisabled"
+                  @click="resetData('log')"
```
Comment (sujit-h-singh) [inline manager/gui/src/components/configuration/LrClusterConfiguration.vue:161]: confirmation modal required

**Review by sujit-h-singh — CHANGES_REQUESTED** [review]

File: manager/gui/src/components/configuration/LrNodeConfiguration.vue (hunk 1 above)
Comment (NareshGopal) [reply manager/gui/src/components/configuration/LrNodeConfiguration.vue]: A confirmation modal has been added.

File: manager/gui/src/components/configuration/LrClusterConfiguration.vue (hunk 2 above)
Comment (NareshGopal) [reply manager/gui/src/components/configuration/LrClusterConfiguration.vue]: A confirmation modal has been added.

**Review by sujit-h-singh — APPROVED** [review]

**Pull request merged**